lib/pay_day_loan/load_worker.ex

```diff
@@ -67,7 +67,16 @@ defmodule PayDayLoan.LoadWorker do
     {:noreply, %{state | load_task_ref: nil}}
   end
 
-  def handle_info({:DOWN, _ref, :process, _pid, _reason}, state) do
+  def handle_info({:DOWN, _ref, :process, _pid, _reason}, %{pdl: pdl} = state) do
+    # The loader process died, so reset the status of any keys that were in the :loading
+    # or :request_loading states to unloaded.
+
+    pending_keys =
+      LoadState.loading_keys(pdl.load_state_manager) ++
+        LoadState.reload_loading_keys(pdl.load_state_manager)
+
+    LoadState.unload(pdl.load_state_manager, pending_keys)
```

Comment:
I don't know if this will have a signifiicant imact, but doing ++ is slow if you have two large lists. I might do
```suggestion
    pending_keys =
      LoadState.loading_keys(pdl.load_state_manager)
    LoadState.unload(pdl.load_state_manager, pending_keys)

    pending_keys =
      LoadState.reload_loading_keys(pdl.load_state_manager)
    LoadState.unload(pdl.load_state_manager, pending_keys)
```

Reply:
did basically the same thing, but without the reassigned variable